### ISSUE TITLE
Action port: Add support for custom project fields

### ISSUE THREAD

**Issue by Bullrich** [paritytech/github-issue-sync#44]
Add support for [custom project fields](https://docs.github.com/en/issues/planning-and-tracking-with-projects/understanding-fields).

Add the ability to add extra values when moving the issue to a board.

Must be done after #43

**Comment by Bullrich** [comment]
@paritytech/opstooling @mordamax does it makes sense to develop this task or should we guide the users to actually use [the built in project workflow automations](https://docs.github.com/en/issues/planning-and-tracking-with-projects/automating-your-project/using-the-built-in-automations)?

As they can set it up so everytime a new issue is added it automaticall assigns the field based on their configuration:
<img width="447" alt="image" src="https://user-images.githubusercontent.com/8524599/212963047-bcdd9bd6-da11-46dd-bad0-4b0c56ba85f2.png">

Please advise.
